gpflow/models/model.py

```diff
@@ -333,7 +333,7 @@ def predict_log_density(
         self, data: RegressionData, full_cov: bool = False, full_output_cov: bool = False
     ) -> tf.Tensor:
         """
-        Compute the log density of the data at the new data points.
+        Compute the log of the probability density of the data at the new data points.
```

Comment:
Genuine question: is log-density not a standard term?

Reply:
I think it's standard enough, but it doesn't hurt to be a bit verbose in the docstring. The function is already called log_density..